docs/setup/kubeedge_install_source.md

```diff
@@ -13,7 +13,9 @@ git clone https://github.com/kubeedge/kubeedge.git $GOPATH/src/github.com/kubeed
 cd $GOPATH/src/github.com/kubeedge/kubeedge
 ```
 
-### Generate Certificates
+### Generate Certificates(This step is required except kubeedge1.3)
+
+**Note: Kubeedge1.3 needs to skip this step Generate Certificates and clean up the local certificates in `/etc/kubeedge/ca` and `/etc/kubeedge/certs`. Because kubeedge1.3 has added the feature of generating certificates automatically.**
```

Comment:
```suggestion
**Note: KubeEdge v1.3 needs to skip this step Generate Certificates and clean up the local certificates in `/etc/kubeedge/ca` and `/etc/kubeedge/certs`. Because KubeEdge v1.3 has added the feature of generating certificates automatically.**
```